src/S3Policy.js

```diff
@@ -35,7 +35,9 @@ export class S3Policy {
 }
 
 const getDate = () => {
-  let date = new Date();
+  let date = new Date(
```

Comment:
I meant revert the current code back to what it was.

Right now it's:

```javascript
let date = new Date(
  (new Date).getTime()
);
```

But it was previously 

```javascript
let date = new Date();
```

This removes the change and minimizes the diff.